panel/lxqtpanel.cpp

```diff
@@ -374,7 +372,7 @@ int LXQtPanel::getReserveDimension()
     return mHidable ? PANEL_HIDE_SIZE : qMax(PANEL_MINIMUM_SIZE, mPanelSize);
 }
 
-void LXQtPanel::setPanelGeometry(bool animate)
+void LXQtPanel::setPanelGeometry(bool animate, bool firstTime)
```

Comment:
Yes, the "flash" is gone now. However:

With your first suggestion, the animated panels will be recognized once they start to hide. So, it seems a *frugal* way of implementing panel animation and I willingly chose to ignore my obsession with start-up animation because of *that*.

With this suggestion, the initial animation is done well but at the cost of a simpler (or at least, easier to understand) code.